test/settings/DynamoSettings-NewSettings.xml

```diff
@@ -8,6 +8,7 @@
   <ShowCodeBlockLineNumber>false</ShowCodeBlockLineNumber>
   <ShowConnector>false</ShowConnector>
   <ShowConnectorToolTip>false</ShowConnectorToolTip>
+  <LibraryZoomScale>2.0361751152073739</LibraryZoomScale>
```

Comment:
Is the value intended to be so long? Can we limit the digit length of the value serialized somehow?

Reply:
We can use float instead of double

Reply:
Works for me